Extension/lib/content-script/assistant/js/adguard-rules-constructor.js

```diff
@@ -105,6 +94,21 @@ var AdguardRulesConstructorLib = (function (api) {
         return path.join(" > ");
     };
 
+    var getElementClassName = function (element) {
```

Comment:
It will be work wrong, if class contains '.'
For example <div class="good-class bad.class.name"></div>